Utils/AppConstants.swift

```diff
@@ -200,6 +200,23 @@ public struct AppConstants {
         #endif
     }()
 
+    /// Toggles the ability to reorder tabs in the tab tray
+    public static let MOZ_REORDER_TAB_TRAY: Bool = {
+        #if MOZ_CHANNEL_RELEASE
+            return false
+        #elseif MOZ_CHANNEL_BETA
+            return false
+        #elseif MOZ_CHANNEL_NIGHTLY
+            return false
```

Comment:
Oops sorry I misread and thought this was the tab reordering flag. I think it's probably best to keep this off for now since we don't actually store or use the metadata yet on master.